src/modules/live-execution-tracker/services/live-execution-tracker.service.ts

```diff
@@ -399,9 +412,11 @@ export class LiveExecutionTrackerService implements ILiveExecutionTrackerService
   public async getColorizedDiagram(processInstanceId: string): Promise<string> {
     const elementsWithActiveToken: Array<IShape> = await this.getElementsWithActiveToken(processInstanceId);
     const elementsWithTokenHistory: Array<IShape> = await this.getElementsWithTokenHistory(processInstanceId);
+    const elementsWhereProcessWasStopped: Array<IShape> = await this.getElementsWhereProcessWasStopped(processInstanceId);
```

Comment:
For consistent naming, I'd suggest:

```suggestion
    const elementsWithError: Array<IShape> = await this.getElementsWhereProcessWasStopped(processInstanceId);
```

Remember, a Processs is not necessarily stopped, if an Activity encounters an error.
If the activity has an `ErrorBoundaryEvent` that handles the error, the ProcessInstance will continue on.